docs/insights/code.md

```diff
@@ -534,12 +534,259 @@ This sniff disallow usage of `global`.
 
 **Insight Class**: `PHP_CodeSniffer\Standards\Squiz\Sniffs\PHP\GlobalKeywordSniff`
 
-## Forbiden Globals <Badge text="^1.0"/> <Badge text="Code\Globally" type="warn"/>
+## Forbidden Globals <Badge text="^1.0"/> <Badge text="Code\Globally" type="warn"/>
 
 This sniff detects globals accesses.
 
 **Insight Class**: `NunoMaduro\PhpInsights\Domain\Insights\ForbiddenGlobals`
 
+## Visibility Required <Badge text="^1.8"/> <Badge text="Code\Classes" type="warn"/> <Badge text="configurable"/>
+
+Visibility must be declared on all properties and methods. `abstract` and `final` must be declared before the visibility. `static` must be declared after the visibility.
+
+**Insight Class**: `PhpCsFixer\Fixer\ClassNotation\VisibilityRequiredFixer`
+
+<details>
+    <summary>Configuration</summary>
+
+```php
+\PhpCsFixer\Fixer\ClassNotation\VisibilityRequiredFixer::class => [
+    'elements' => [
+        'property',
+        'method',
+    ],
+]
+```
+</details>
+
+## Ternary to Null Coalescing <Badge text="^1.8"/> <Badge text="Code\Code" type="warn"/>
+
+This sniff enforce use null coalescing operator `??` where possible.
```

Comment:
```suggestion
This sniff enforce using null coalescing operator `??` where possible.
```

Reply:
Should we call the fixers for sniffs or fixers in the docs? This is a case in a lot of places there.

Reply:
> Should we call the fixers for sniffs or fixers in the docs? This is a case in a lot of places there

You are right ! I left `This sniff` in template, but we should change them. Thanks :wink: